R/annotdata.R

```diff
@@ -546,7 +546,7 @@ getPointSegmentsForPPT <- function(points, index, annot, dat)
         # Note that we use i to select elements of dat because it is assumed that we are only
         # passing the relevant sections of the data (i.e. dat is already subsetted by index)
         if (grepl("^Text", annot$type))
-            tmp.seg[[1]]$Text <- formatByD3(dat[i], annot$format, annot$prefix, annot$suffix)
+            tmp.seg[[1]]$Text <- unescape_html(formatByD3(dat[i], annot$format, annot$prefix, annot$suffix))
```

Comment:
Was this an existing issue (prior to the recent work for column comparisons)? For example if there was an ampersand in the label, it would appear as `&#x26;`?

Reply:
The `unescape_html` function existed previously to deal with the arrow html entities, but we never added applied it to the text annotations. I'm not really sure what users put in their text annotations but I guess adding some additional characters in the test can't hurt.